examples/footer/index.html

```diff
@@ -8,59 +8,85 @@
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/linking-policy.html" title="Linking Policy">Linking Policy</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-privacy-practices.html" title="Notice of Privacy Practices (HIPAA)">Notice of Privacy Practices (HIPAA)</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-non-discrimination.html" title="Notice of Non-Discrimination">Notice of Non-Discrimination</a>
-          </li>
-        </ul>
-        <ul class="social-media">
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/stjuderesearch" class="fab fa-facebook-f">
-              <span class="sr-only">Facebook</span>
+    <footer>
+      <div class="container">
+        <div class="row">
+          <div class="col-6 col-lg-3 col-md-4 offset-xl-0 offset-lg-5 offset-md-4 offset-md-3 offset-3 footer-logo">
+            <a class="navbar-brand" href="https://stjude.cloud">
+              <span class="logo"></span>
             </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/StJudeResearch" class="fab fa-twitter">
-              <span class="sr-only">Twitter</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.instagram.com/stjuderesearch/" class="fab fa-instagram">
-              <span class="sr-only">Instagram</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/company/st-jude-childrens-research-hospital/" class="fab fa-linkedin">
-              <span class="sr-only">LinkedIn</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/user/MyStJude" class="fab fa-youtube">
-              <span class="sr-only">YouTube</span>
-            </a>
-          </li>
-        </ul>
+            <a class="sjc-title" href="https://stjude.cloud">St. Jude Cloud</a>
+          </div>
+          <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-about">
+            <h6>About St. Jude</h6>
+            <ul>
+              <li>
+                <a href="https://www.stjude.org">Home</a>
+              </li>
+              <li>
+                <a href="https://www.stjude.org/about-st-jude.html">About</a>
+              </li>
+              <li>
+                <a href="https://stjude.org/contact-us.html">Contact</a>
+              </li>
+            </ul>
+          </div>
+          <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-apps">
+            <h6>St. Jude Cloud Apps</h6>
```

Comment:
@jamesmadson do you have a vision for what adding an extra app will look like? I know we only have three now (which lines up nice with the other column) but could see that number growing in the not so distant future.

Reply:
I think we'll address that when we come to it. As it sits right now, the columns would likely be ok with up to 6 apps listed. We'd maybe need to see how the apps are categorized (if at all) on the main SJ Cloud site.